test/integration/admin/api/registry/policies_controller_test.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class Admin::Api::Registry::PoliciesControllerTest < ActionDispatch::IntegrationTest
+  def setup
+    @provider = FactoryBot.create(:provider_account)
+    host! @provider.admin_domain
+    @access_token = FactoryBot.create(:access_token, owner: @provider.admin_users.first!, scopes: %w[account_management], permission: 'rw')
+  end
+
+  test 'POST create with valid params persists with those values' do
+    assert_difference(@provider.policies.method(:count), 1) do
+      post admin_api_registry_policies_path(policy_params)
+    end
+    assert_response :created
+    assert JSON.parse(response.body).dig('policy', 'id')
+    policy = @provider.policies.last!
+    policy_params[:policy].each { |key, value| assert_equal(value, policy.public_send(key)) }
```

Comment:
Let me understand this assertion. By posting `policy_params` it creates a new policy, then it verifies the last provider's policy is equal to the one posted.

Reply:
It comes from https://github.com/3scale/porta/blob/053897a3f3b696e91957e6daf42d99d44eea2f82/test/integration/admin/api/registry/policies_controller_test.rb#L81

Reply:
This verifies that what was passed as parameter is stored in the database for: `name`, `version`, schema`

Reply:
This is an integrity check, @josemigallas. It's basically asserting that the record stores the same values originally passed as parameters.